packages/compass-components/src/components/resizeable-sidebar.tsx

```diff
@@ -58,30 +58,20 @@ const containerStylesLight = css({
 export const defaultSidebarWidth = spacing[6] * 4;
 
 const ResizableSidebar = ({
-  collapsable = false,
-  expanded = true,
-  setExpanded = () => {
-    return;
-  },
   initialWidth = defaultSidebarWidth,
   minWidth = 210,
-  collapsedWidth = 48,
+  // TODO: maxWidth
```

Comment:
actually I'll just remove this line before merging. But I'm leaving CI to run for the moment